src/store/flow/flow.ts

```diff
@@ -16,20 +17,69 @@ import moment from 'moment'
 import { ActionTree, GetterTree, MutationTree } from 'vuex'
 import { IRootState } from '@/store'
 import {
-  defaults, includes, forEach, cloneDeep, get, has,
+  defaults, 
+  includes, 
+  forEach, 
+  cloneDeep, 
+  get, 
+  has, 
+  omit
 } from 'lodash'
 import { discoverContentTypesFor } from '@/store/flow/resource'
 import { computeBlockPositionsFrom } from '@/store/builder'
 import { IFlowsState } from '.'
 
 export const getters: GetterTree<IFlowsState, IRootState> = {
-  activeFlow: (state) => state.flows.length && getActiveFlowFrom(state as unknown as IContext),
-
+  //We allow for an attempt to get a flow which doesn't yet exist in the state - e.g. the firstFlowId doesn't correspond to a flow
+  activeFlow: (state) => {
+    if(state.flows.length) { 
+      try {
+        return getActiveFlowFrom(state as unknown as IContext)
+      } catch(err) {
+        return 0
+      }
+    }
+  },
+  //TODO - is the IContext equivalent to the Flow Container? Can we say that it should be?
+  activeFlowContainer: state => {
+    return {
+      created: state.created,
+      specification_version: "TODO",
+      uuid: "TODO",
+      name: "TODO",
+      description: "TODO",
+      platform_metadata: {},
+      flows: state.flows,
+      resources: state.resources
+    } as unknown as IContext
+  },
+  flowsList: state => {
+    return state.flows
```

Comment:
Out of curiosity: Would you prefer to keep this getter rather than removing + doing a `mapState` directly in the component that consumes `flowsList` ? 

The reason I'm asking is that it feels like a tiny bit of added indirection in a space where we haven't reconciled `IContext`, `IContainer`, `activeFlowContainer` (above) — Might be nice to mitigate having yet another way to find a list of flows 🤷‍♀️ 😸

Reply:
Hmm. I think I get what you are saying. The Getter isn't necessary yet so let's move to mapState.